spec/javascript/components/applications/__snapshots__/ApplicationEditPage.test.js.snap

```diff
@@ -1451,7 +1451,7 @@ Array [
                             onBlur={[Function]}
                             onChange={[Function]}
                             onFocus={[Function]}
-                            value="Child"
```

Comment:
I'm a bit concerned about these big snapshot changes. do you know where this one is coming from?

Reply:
This is fixed, this was due to alt contacts being listed as household members